examples/gno.land/r/gh/oracle.gno

```diff
@@ -0,0 +1,102 @@
+package gh
+
+import (
+	"std"
+	"strings"
+	"time"
+
+	"gno.land/p/demo/avl"
+)
+
+var (
+	accounts       avl.Tree    // uri -> Account
+	repos          avl.Tree    // uri -> Repo
+	issueOrPRs     avl.Tree    // uri -> IssueOrPR
+	lastUpdateTime time.Time   // used by the bot to only upload the diff
+	oracleAddr     std.Address = "g1eunnckcl6r8ncwj0lrpxu9g5062xcvwxqlrf29"
+	adminAddr      std.Address = "g1u7y667z64x2h7vc6fmpcprgey4ck233jaww9zq" // @manfred
+)
+
+func OracleLastUpdated() time.Time { return lastUpdateTime }
+
+func OracleUpsertAccount(id, name, kind string) {
```

Comment:
Upsert feels sql-y, when under the hood we just have a k-v. Can we not do `Set`?